### PR TITLE
wifi: make it work on nets with no password

### PR DESCRIPTION
**Pull request by rminnich**
Tested at SC17 where there is no password.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

File: cmds/wifi/wifi.go [hunk 1]
```diff
@@ -6,18 +6,28 @@ package main
...
+	key_mgmt=NONE
+}
+`
+)
 
 var (
 	iface = flag.String("i", "wlan0", "interface to use")
 	essid string
 	pass  string
+	conf  []byte
```
Comment (hugelgupf) [inline cmds/wifi/wifi.go]: all three of these could just be local variables in main?

Comment (hugelgupf) [reply cmds/wifi/wifi.go]: other than that, looks all good.

Comment (rminnich) [reply cmds/wifi/wifi.go]: great suggestion, done.

File: cmds/wifi/wifi.go [hunk 2]
```diff
@@ -29,20 +32,32 @@ func init() {
 }
 
 func main() {
+	var (
+		iface = flag.String("i", "wlan0", "interface to use")
```
Comment (hugelgupf) [inline cmds/wifi/wifi.go]: well, the flag can be global :) i was just talking about the other three. but either way, it's a small program, so whatever.

Comment (rminnich) [reply cmds/wifi/wifi.go]: yeah, it can be made global should it ever matter, which at the moment it does not :-)

**Pull request merged**